src/node/util.ts

```diff
@@ -482,7 +482,7 @@ export function escapeHtml(unsafe: string): string {
  * it has a .code property.
  */
 export function isNodeJSErrnoException(error: unknown): error is NodeJS.ErrnoException {
-  return error instanceof Error && (error as NodeJS.ErrnoException).code !== undefined
+  return error !== undefined && (error as NodeJS.ErrnoException).code !== undefined
```

Comment:
@code-asher and I learned `ErrnoException` extends `Error` but it is not an instance of `Error` so we had to fix this check

Reply:
Thoughts on this approach: https://stackoverflow.com/a/51533259 ?

Reply:
> Thoughts on this approach: https://stackoverflow.com/a/51533259 ?

Maybe I'm misunderstanding the suggested solution, but I believe that's exactly why this change is necessary. I don't think that works 😢

> I learned ErrnoException extends Error but it is not an instance of Error

The typeguard works so it comes back as `NodeJS.ErrnoException` which is correct but using `error instanceof Error` when passing in `NodeJS.ErrnoException` returns `false` because an error of type `NodeJS.ErrnoException` is not an instance of `Error`. 

But maybe I'm missing something?

Reply:
Hmmm...there might be something more at play.  Check out what I get on my machine:

```
$ node -e 'try { fs.readFile("/") } catch (error) { console.log(error instanceof Error) }'
true
```

What do you get?

Reply:
Oh wow....Nice find! Hmm...how should we handle this then?

Reply:
I am not sure...options are to switch off Jest, look to see if there is a fix in progress (maybe this one? https://github.com/facebook/jest/pull/8220 but it is rather old), or patch it ourselves.

Reply:
Maybe I'll make an issue and do this in a followup PR if that's okay with y'all.

Reply:
Sure thing.

Reply:
Tracking here: https://github.com/cdr/code-server/issues/4303